From 24839cd95cc50bbbebe269c932cc4a023a1ebef1 Mon Sep 17 00:00:00 2001
Message-ID: <24839cd95cc50bbbebe269c932cc4a023a1ebef1.1779975966.git.bertrand.marquis@arm.com>
In-Reply-To: <55d56a174a0f04520d4a1558630bb0b739074b13.1779975966.git.bertrand.marquis@arm.com>
References: <55d56a174a0f04520d4a1558630bb0b739074b13.1779975966.git.bertrand.marquis@arm.com>
From: Bertrand Marquis <bertrand.marquis@arm.com>
Date: Thu, 16 Apr 2026 19:30:48 +0200
Subject: [PATCH 30/30] configs/devices: add vmsg-bridge-min profile

Add the minimal device profile for the virtio-msg bridge configuration.

Keep this as the final top-of-series fix/config patch.

Signed-off-by: Bertrand Marquis <bertrand.marquis@arm.com>
---
 .../devices/aarch64-softmmu/vmsg-bridge-min.mak    | 14 ++++++++++++++
 1 file changed, 14 insertions(+)
 create mode 100644 configs/devices/aarch64-softmmu/vmsg-bridge-min.mak

diff --git a/configs/devices/aarch64-softmmu/vmsg-bridge-min.mak b/configs/devices/aarch64-softmmu/vmsg-bridge-min.mak
new file mode 100644
index 000000000000..edbe3a6b769b
--- /dev/null
+++ b/configs/devices/aarch64-softmmu/vmsg-bridge-min.mak
@@ -0,0 +1,14 @@
+CONFIG_ARM_VIRT=y
+CONFIG_PCI_DEVICES=n
+CONFIG_I2C_DEVICES=n
+CONFIG_TEST_DEVICES=n
+
+CONFIG_VIRTIO_MMIO=y
+CONFIG_VIRTIO_MSG=y
+CONFIG_VIRTIO_MSG_LINUX_BRIDGE=y
+CONFIG_VIRTIO_MSG_AMP_PCI=n
+
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_NET=y
+CONFIG_VIRTIO_RNG=y
+CONFIG_VIRTIO_SERIAL=y
-- 
2.53.0

